frontend/src/pages/OrganizationProfile.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from 'react';
+import { OrderDirection } from '../common/enums/order-direction.enum';
+import { IPaginatedEntity } from '../common/interfaces/paginated-entity.interface';
+import Divisions, { DivisionTabs, DivisionType, IDivision } from '../components/Division';
+import { useErrorToast } from '../hooks/useToast';
+import PageLayout from '../layouts/PageLayout';
+import i18n from '../common/config/i18n';
+import { SortOrder, TableColumn } from 'react-data-table-component';
+import { useDivisionDataQuery } from '../services/division/division.service';
+
+const OrganizationProfile = () => {
+  const [divisionType, setDivisionType] = useState<DivisionType>(DivisionType.Branches);
+  const [data, setData] = useState<IPaginatedEntity<IDivision>>();
```

Comment:
here we can be more specific in what kind of data it this variable describes, we can call it divisions